app-office/libreoffice/libreoffice-4.4.9999.ebuild

```diff
@@ -360,26 +335,35 @@ src_prepare() {
 		epatch
 	fi
 
-	base_src_prepare
+	# not upstreamable stuff
+	epatch "${FILESDIR}/${PN}-4.4-system-pyuno.patch"
+
+	# from 5.0 branch
+	epatch "${FILESDIR}/${PN}-4.3.5.2-remove-bashisms.patch" \
+		"${FILESDIR}/${PN}-4.4.0.3-telepathy-build-fix.patch" \
+		"${FILESDIR}/${PN}-4.4.1.2-add-kde4-open-url-script.patch" \
+		"${FILESDIR}/${PN}-4.4.4.3-improve-KDE4FilePicker.patch" \
+		"${FILESDIR}/${PN}-4.4.4.3-fix-KDE4-FileDialog.patch" \
+		"${FILESDIR}/${PN}-4.4.4.3-cleanup-IsNativeControlSupported.patch"
+
+	# from master branch
+	epatch "${FILESDIR}/${PN}-4.4.5.2-fix-KDE4-listbox-regression.patch"
 
 	AT_M4DIR="m4" eautoreconf
 	# hack in the autogen.sh
 	touch autogen.lastrun
 
 	# system pyuno mess
-	sed \
-		-e "s:%eprefix%:${EPREFIX}:g" \
+	sed -e "s:%eprefix%:${EPREFIX}:g" \
 		-e "s:%libdir%:$(get_libdir):g" \
 		-i pyuno/source/module/uno.py \
 		-i pyuno/source/officehelper.py || die
```

Comment:
Yes! Found something to nitpick! Second `-i` is redundant, since the option affects all files.

I usually do it like this but others have different preferences:

    sed -i \
        -e ... \
        -e ...